### PR TITLE
AV-193307: Updated error code and added default values for error scenarios

### PR DESCRIPTION
**Pull request by abhineshgour**
AV-193307: Updated error code and added default values for error scenarios

### CODE REVIEW

File: server/app/controllers/migration.controller.js [hunk 1]
```diff
@@ -356,7 +356,7 @@ const updateStatus = async (req, res) => {
                 }});
             }
         } else {
-            res.status(404).json({ error: "Conversion Status Profile you are trying to update does not exists" });
+            res.status(200).json({ message: "Profile you are trying to update does not exists." });
```
Comment (kumarsuraj27) [inline server/app/controllers/migration.controller.js:359]: Should it not be 500 for that

File: server/app/controllers/migration.controller.js [hunk 2]
```diff
@@ -307,7 +307,7 @@ const updateAviObjectInAviOutputModel = async (req, res) => {
             const updateQuery = `${profileType}.${indexToUpdate}`;
             await AviOutputModel.findOneAndUpdate(findQuery, { $set: { [updateQuery]: aviObjectToUpdate } });
         } else {
-            res.status(404).json({ error: "Profile you are trying to update does not exists" });
+            res.status(200).json({ message: "Profile you are trying to update does not exists." });
```
Comment (kumarsuraj27) [inline server/app/controllers/migration.controller.js:310]: Should it not be 500 for that

File: server/app/controllers/migration.controller.js (hunk 1 above)
Comment (abhineshgour) [reply server/app/controllers/migration.controller.js:359]: 500 we are only sending when there is some error, that is mostly the runtime error, for this case the query ran successfully on DB but the data on the query is not available. So we are updating Client accordingly.

Comment (kumarsuraj27) [reply server/app/controllers/migration.controller.js:359]: Yes as we discussed about status and UI code, update it with 500 for that

**Review by kumarsuraj27 — APPROVED** [review]

File: server/app/controllers/migration.controller.js (hunk 2 above)
Comment (abhineshgour) [reply server/app/controllers/migration.controller.js]: updated

File: server/app/controllers/migration.controller.js (hunk 1 above)
Comment (abhineshgour) [reply server/app/controllers/migration.controller.js]: updated

**Pull request merged**